spec/turn_spec.rb

```diff
@@ -9,22 +9,22 @@
     end
 
     it "has a player" do
-      player = Player.new
+      player = Player.new("X")
```

Comment:
Not a big deal, but to slightly shorten up the test suite we could "before" the player before the tests since it's in very test.